scripts/exporter.py

```diff
@@ -323,6 +324,62 @@ def main(self) -> None:
         raise NotImplementedError("Marching cubes not implemented yet.")
 
 
+@dataclass
+class ExportSDFMarchingCubesMesh(Exporter):
```

Comment:
Can we make this generic and not specific to an SDF? I think it would be better for nerfstudio. We can specify the isosurface by a threshold that is set as a variable in this function as opposed to assuming it is zero. This way we can do marching cubes on NeRF densities too.